test/components/user_list_item_test.js

```diff
@@ -45,6 +59,57 @@ describe("UserListItem", () => {
     })
   })
 
+  describe("passed a non-facilitator as a facilitator", () => {
+    const user = { ...defaultUserAttrs, is_facilitator: true }
+    const user2 = { ...secondaryrUserAttrs, is_facilitator: false }
+
+    it("renders the magic icon", () => {
+      const wrapper = shallow(
+        <UserListItem {...defaultProps} user={user2} currentUser={user} />
+      )
+      expect(wrapper.find("i.magic.icon")).to.have.length(1)
+    })
+
+    it("renders facilitator button", () => {
+      const wrapper = shallow(
+        <UserListItem {...defaultProps} user={user2} currentUser={user} />
+      )
+      expect(wrapper.find("button.facilitator")).to.have.length(1)
+    })
+
+    describe("passing facilitatorship", () => {
+      beforeEach(() => {
+        window.confirm = sinon.stub().returns(true)
+      })
+
+      it("calls the action to change facilitator", () => {
+        const wrapper = shallow(
+          <UserListItem {...defaultProps} user={user2} currentUser={user} />
+        )
+        wrapper.find("button.facilitator").simulate("click")
+        expect(defaultProps.actions.updateRetroAsync.calledWith({ facilitator_id: user2.id }))
```

Comment:
same as above. curious if this will always fail, as there is no assertion tagged on the `expect()`?